### PR TITLE
feat: Adjust skr watcher image flag

### PR DESCRIPTION
**Pull request by nesmabadr**
**Description**

Changes proposed in this pull request:

- Adjust runtime watcher image flag usage
- Adapt watcher local setup to use the flag

**Related issue(s)**
#919

### CODE REVIEW

File: config/watcher_local_test/kustomization.yaml [hunk 1]
```diff
@@ -37,6 +37,9 @@ patches:
     - op: add
       path: /spec/template/spec/containers/0/args/-
       value: --listener-port-overwrite=9443
+    - op: add
+      path: /spec/template/spec/containers/0/args/-
+      value: --skr-watcher-image=runtime-watcher-skr:latest
```
Comment (lindnerby) [inline config/watcher_local_test/kustomization.yaml:42]: this should be the same as in `skrwebhook/resources.yaml` right?

Comment (lindnerby) [reply config/watcher_local_test/kustomization.yaml:42]: or do we leave latest here and for the config/default we anyways overwrite with tag?

Comment (lindnerby) [reply config/watcher_local_test/kustomization.yaml:42]: As discussed let's keep testing latest here 👍

**Review by lindnerby — DISMISSED** [review]

**Review by lindnerby — APPROVED** [review]

**Review by ruanxin — APPROVED** [review]

**Review by ruanxin — APPROVED** [review]

**Pull request merged**